src/class-wc-gateway-btcpay.php

```diff
@@ -838,22 +851,27 @@ public function process_payment($order_id)
             );
         }
 
-        public function ipn_callback()
+        //https://<host>/wp-json/btcpay/ipn/status
+        function btcpay_checkout_ipn(WP_REST_Request $request)
         {
             $this->log('    [Info] Entered ipn_callback()...');
-            // Retrieve the Invoice ID and Network URL from the supposed IPN data
-            $post = file_get_contents("php://input");
-
-            if (true === empty($post)) {
+            
+            global $woocommerce;
+            WC()->frontend_includes();
```

Comment:
what is happening here now?

Reply:
good catch, that's old code. Can be removed. I'll create a new PR